### DISCUSSION TITLE
latest vs unstable

### DISCUSSION THREAD

**Discussion by tamimology** [CyferShepard/Jellystat#155] [Q&A]
I have pulled both the latest and unstable tags, why the latest is double in size?

![image](https://github.com/CyferShepard/Jellystat/assets/90429593/e127eb87-133a-42f3-a9ed-25f085fc4388)

However, both tags are showing the same version in the UI

![image](https://github.com/CyferShepard/Jellystat/assets/90429593/dcf7c284-a1a6-4421-8d71-88faea2e0698)

The latest was updated 5 days ago and unstable 3 days.

![image](https://github.com/CyferShepard/Jellystat/assets/90429593/31ae56e7-bd7b-43f6-add6-8c3fc9a45304)

One last thing I noticed, the logs of the last have the following once loaded (unstable does not)
```
> jfstat@1.0.9 start
> cd backend && node server.js
```

Honestly, both are working as a charm for me now, but wanted why to choose one over another especially since the size is double. Any features are added in the latest?

**Comment by CyferShepard** [comment]
Hey @tamimology , im not really sure why the 2 have different sizes, they should be around the same but i can look into that.

In regards to the version number its just that i sometimes forget to bump the version number on unstable, i normally only bump it closer to when i plan to merge everything back into the main branch.

Technically both are pretty similar, its just that unstable will have changes/improvements/new features from the development branch with potentially breaking bugs
